Doc/library/asyncio-task.rst

```diff
@@ -414,53 +433,6 @@ reported by :meth:`asyncio.Task.cancelling`.
    Improved handling of simultaneous internal and external cancellations
    and correct preservation of cancellation counts.
 
-Terminating a Task Group
```

Comment:
These docs make sense for older versions.

Reply:
Probably recommending a backport module would be better

Reply:
@belm0 are you interested in applying this change and any previous changes to my taskgroup backport?

Reply:
I'm talking about backporting to pypi